app/helpers/automated_tests_client_helper.rb

```diff
@@ -252,13 +251,11 @@ def self.request_a_test_run(host_with_port, grouping_id, call_on, current_user,
   def self.get_test_scripts_to_run(assignment, call_on)
 
     all_scripts = TestScript.where(assignment_id: assignment.id)
-    # If the test run is requested at collection (by Admin or TA),
+    # If the test run is requested after collection (by Admin or TA),
     # All of the test scripts should be run.
-    if call_on == 'collection'
+    if call_on == 'after_collection'

```

Comment:
@adisandro maybe I wasn't clear about this.

It is true that we want to differentiate between scripts run for students vs. ones run for instructors. However, it should be possible to mark scripts which are run *only* by students, and not for instructors. This is why we need the `run_on_submission` attribute; it and `run_on_request` aren't strictly negations of each other.